### PR TITLE
fix bottom space on forYou's NewsFeed last item 

### LINKED ISSUE #204: [Bug]: forYou's NewsFeed has no space at the bottom of the last item
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is there a StackOverflow question about this issue?

- [X] I have searched StackOverflow

### What happened?

The NewsFeed of forYou does not leave spaces at the bottom of the last item.

if spaces are added, the user's visual experience may be better.

Consider simply adding contentPadding = PaddingValues(bottom = 16.dp).

### Relevant logcat output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by YvesKalume**
fix #204  
I have resolved the issue of merging previous PR #215 with fixed issues

### CODE REVIEW

File: feature-foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt [hunk 1]
```diff
@@ -203,8 +205,13 @@ fun ForYouScreen(
                     loadingContentDescription = R.string.for_you_loading
                 )
 
-                item(span = { GridItemSpan(maxLineSpan) }) {
```
Comment (alexvanyo) [inline feature-foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt:206]: Ah sorry, this wasn't a clean merge. We changed to use a `LazyGrid` view here, and therefore we want the spacing items to fill up the entire span (all columns, if there are more than one).

That ensures that this spacing is applied correctly, as opposed to being an item at the end of a row after an item that doesn't fill up all of the space.

Let's add the `span = { GridItemSpan(maxLineSpan) }` back in to the `item`s. We also probably could have just one `item` instead, and have a `Column` of the two `Spacer`s.

Comment (YvesKalume) [reply feature-foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt:206]: Done ! Can you check ?

**Review by alexvanyo — APPROVED** [review]
Looks good, sorry for the merge conflicts and thank you for updating!

**Comment by YvesKalume** [comment]
Now that the conflict is resolved, I hope this will be merged before there is another 😅

**Review by JolandaVerhoef — APPROVED** [review]

**Pull request merged**